ui/webpack.config.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+
+// check/detect if a production build (or not) via the webpack -p flag
+const IS_PRODUCTION = process.argv.indexOf('-p') != -1;
+
+const parentDir = path.join(__dirname, './');
+const PUBLIC_DIR = path.resolve(__dirname, './public/');
+const BUILD_DIR = path.resolve(__dirname, './dist')
```

Comment:
Just to confirm - this would be inside of `src` currently? Ideally (not that I have said this yet) I would want the dist as a peer of `src` - just to keep source and built content seperate

Reply:
Same case here. `dist`, `src`, and `public` are all at the same level

Reply:
Cool - happy to debate it - but I see the public assets as source code, and thus they should like in src. Built output is not, hence should not be in src. Thoughts?

Reply:
While I agree the public folder is source code, the src folder seems most relevant for the JS/SCSS sources as that's what the app is made of. I feel it would be out of place amongst all that.

Reply:
I agree - JS/(raw) SCSS should absolutley be in in `src`. To confirm - the structure we are talking about is:
```
src/
dist/
public/
```
..right?

If so - my second comment was more for moving public in to `src` too (and discussion around that).

Reply:
Yep! I see the public folder more closely related to webpack than the actual source code so I don't think it would fit in being inside the `src` folder

Reply:
Fair point - it has other advantages thinking about it being external too (easier to expose for express etc). Cool - think we leave it as is :)